infra/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "digitalocean" {
+  # You need to set this in your .bashrc
+  # export DIGITALOCEAN_TOKEN="Your API TOKEN"
+  #
+}
+
+resource "digitalocean_droplet" "delta_droplet" {
+  ssh_keys           = [8187378]
```

Comment:
This _should_ be ok. Briefly, it's the ID of the public key I generated, stored in DO, and will be provisioned in the default account of the image.

However, it's pretty unclear, so I'll def add a comment.